### PR TITLE
Fix properties of RegExp prototype

### PR DESCRIPTION
**Pull request by szledan**
JerryScript-DCO-1.0-Signed-off-by: László Langó llango.u-szeged@partner.samsung.com

### CODE REVIEW

File: jerry-core/ecma/builtin-objects/ecma-builtin-regexp-prototype.inc.h [hunk 1]
```diff
@@ -40,6 +52,47 @@ OBJECT_VALUE (LIT_MAGIC_STRING_CONSTRUCTOR,
               ECMA_PROPERTY_NOT_ENUMERABLE,
               ECMA_PROPERTY_CONFIGURABLE)
 
+// ECMA-262 v5, 15.10.7.1
+STRING_VALUE (LIT_MAGIC_STRING_SOURCE,
+              LIT_MAGIC_STRING_REGEXP_SOURCE_UL,
```
Comment (ruben-ayrapetyan) [inline jerry-core/ecma/builtin-objects/ecma-builtin-regexp-prototype.inc.h]: Seems that this is not correct value for `RegExp.prototype.source`.

File: jerry-core/ecma/builtin-objects/ecma-builtin-regexp-prototype.inc.h [hunk 2]
```diff
@@ -40,6 +52,47 @@ OBJECT_VALUE (LIT_MAGIC_STRING_CONSTRUCTOR,
...
+// ECMA-262 v5, 15.10.7.5
+NUMBER_VALUE (LIT_MAGIC_STRING_LASTINDEX_UL,
+              0,
+              ECMA_PROPERTY_WRITABLE,
+              ECMA_PROPERTY_NOT_ENUMERABLE,
+              ECMA_PROPERTY_NOT_CONFIGURABLE)
+
+// ECMA-262 v5, 15.10.5
+NUMBER_VALUE (LIT_MAGIC_STRING_LENGTH,
+              2,
```
Comment (ruben-ayrapetyan) [inline jerry-core/ecma/builtin-objects/ecma-builtin-regexp-prototype.inc.h]: Seems that the property shouldn't exist in RegExp.prototype

**Comment by egavrin** [comment]
Is it possible write JS test for this PR?

File: tests/jerry/regexp-construct.js [hunk 3]
```diff
@@ -98,3 +98,28 @@ r3 = new RegExp(foo)
 
 assert (r1.source === r2.source);
 assert (r2.source === r3.source);
+
+/* RegExp properties */
+assert (RegExp.length === 2);
+assert (RegExp.prototype.source === "");
```
Comment (ruben-ayrapetyan) [inline tests/jerry/regexp-construct.js]: `print ((new RegExp('')).source);` outputs '(?:)'.

Shouldn't the value be the same?

Comment (szledan) [inline tests/jerry/regexp-construct.js]: You are right, I'll fix it.

**Comment by egavrin** [comment]
Test was added. Good to me, now.

**Comment by ruben-ayrapetyan** [comment]
Looks good to me

**Comment by kkristof** [comment]
Looks good to me.

**Pull request merged**